`dist/cli.js` missing after installing `19.0.0`

### Preflight Checklist

* [x] I have read the [contribution documentation](https://github.com/electron/packager/blob/main/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [code of conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project follows, as appropriate.
* [x] I have searched the issue tracker for a bug that matches the one I want to file, without success.

### Issue Details

* **Electron Packager Version:**
  * 19.0.0
* **Electron Version:**
  * 38.3.0
* **Operating System:**
  * Windows 11 Enterprise 24H2 OS build 26100.6899
* **Last Known Working Electron Packager version:**:
  * 18.4.4

### Expected Behavior

`electron-packager ...` should run

### Actual Behavior
`electron-packager ...` doesn't run. It immediately errors with:

```sh
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '...\node_modules\@electron\packager\dist\cli.js' imported from ...\node_modules\@electron\packager\bin\electron-packager.mjs
    at finalizeResolution (node:internal/modules/esm/resolve:274:11)
    at moduleResolve (node:internal/modules/esm/resolve:859:10)
    at defaultResolve (node:internal/modules/esm/resolve:983:11)
    at #cachedDefaultResolve (node:internal/modules/esm/loader:731:20)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:708:38)
    at ModuleLoader.getModuleJobForImport (node:internal/modules/esm/loader:310:38)
    at ModuleJob._link (node:internal/modules/esm/module_job:183:49) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///.../node_modules/@electron/packager/dist/cli.js'
}
```

I noticed that `node_modules/@electron/packager/dist` isn't being installed

### To Reproduce

- Any project, install @electron/packager 19.0.0
- Notice that `node_modules/@electron/packager/dist` doesn't exist, and there's no `cli.js` to be found
- Try running `electron-packager help` via a script in your `package.json`
- Error ensues

--- a/package.json
+++ b/package.json
@@ -85,7 +85,7 @@
     "build": "tsc",
     "build:docs": "npx typedoc",
     "lint": "prettier --check \"*.{ts,js,json}\" && eslint src --cache",
-    "prepublish": "npm run build",
+    "prepack": "npm run build",
     "pretest": "npm run build",
     "test": "vitest run",
     "prepare": "husky install"

